.github/workflows/test_macos.yml

```
@@ -24,7 +24,7 @@ jobs:
     - name: Install git-annex
       run: |
         eval source tools/ci/install-annex.sh ${{ matrix.install_scenario }}
-        echo "::set-env name=PATH::$PATH"
+        echo "PATH=$PATH" >> $GITHUB_ENV
```

```suggestion
        echo "PATH=$PATH" >> "$GITHUB_ENV"
```
just to be on a safer side ;)